examples/tutorials/simple_workgraph.ipynb

```diff
@@ -0,0 +1,507 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4cbfa7f6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! verdi process list -a"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a100106f",
+   "metadata": {},
+   "source": [
+    "Using the pk, we can print the output from verdi"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "01b7e76a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! verdi calcjob res [pk]  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "33dcefc9",
+   "metadata": {},
+   "source": [
+    "# Geometry Optimisation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ddf8a780",
+   "metadata": {},
+   "source": [
+    "The calculations can be setup with WorkGraph very similarily in most cases, in this example we will setup a geometry optimisation calculation. First we setup the the inputs and load the calculation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "95278dce",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiida.orm import Str, Float, Bool\n",
+    "inputs = {\n",
+    "        \"code\": code,\n",
+    "        \"model\": model,\n",
+    "        \"struct\": structure,\n",
+    "        \"arch\": Str(model.architecture),\n",
+    "        \"precision\": Str(\"float64\"),\n",
+    "        \"device\": Str(\"cpu\"),\n",
+    "        \"fmax\": Float(0.1), \n",
+    "        \"opt_cell_lengths\": Bool(False), \n",
+    "        \"opt_cell_fully\": Bool(True), \n",
+    "        \"metadata\": {\"options\": {\"resources\": {\"num_machines\": 1}}},\n",
+    "    }"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1715e25e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiida.plugins import CalculationFactory\n",
+    "geomoptCalc = CalculationFactory(\"mlip.opt\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "17deb40c",
+   "metadata": {},
+   "source": [
+    "We can then start building the WorkGraph similar to before"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9a1095a8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiida_workgraph import WorkGraph\n",
+    "\n",
+    "wg = WorkGraph(\"Geometry Optimisation\")\n",
+    "\n",
+    "geomopt_calc = wg.add_task(\n",
+    "    geomoptCalc,\n",
+    "    name=\"geomopt_calc\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "62cdb8e2",
+   "metadata": {},
+   "source": [
+    "This time if we want to pass the inputs/otuputs directly to the workgraph we can set them manually: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1ba44195",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wg.tasks.geomopt_calc.set(\n",
```

Comment:
Is this the same as setting `wg.inputs`? It's probably worth generally being slightly more explicit that it's not just "directly" setting things, but defining things on a workgraph level rather than a task level.

Reply:
Ok, I see what you mean now.

This looks more like showing an alternative way to set the task-level inputs (rather than doing it when the task is created as we do for single point), not wg-level inputs, but it's still useful to see, and I think the explanation is enough.